torch_geometric/loader/temporal_dataloader.py

```diff
@@ -26,13 +29,44 @@ def __init__(self, data: TemporalData, batch_size: int = 1, **kwargs):
 
         self.data = data
         self.events_per_batch = batch_size
-
+        self.device = device
+        self.negative_sampling = negative_sampling
+        if self.negative_sampling:
+            # Ensure to only sample actual destination nodes as negatives.
+            self.min_dst_idx, self.max_dst_idx = int(self.data.dst.min()), int(
+                self.data.dst.max())
+        self.neighbor_loader = LastNeighborLoader(data.num_nodes,
```

Comment:
I don't think this is correct. In the example, you will create multiple temporal loaders, each with there own neighbor loader. However, you wanna share the neighbor loader state across all of them.